test_runner/src/main/kotlin/ftl/util/StepOutcome.kt

```diff
@@ -9,4 +9,15 @@ object StepOutcome {
     const val skipped = "skipped"
     const val success = "success"
     const val unset = "unset"
+
+    val notErrors = listOf(failure, flaky, skipped, success)
+
+    val order = linkedSetOf(
+        failure,
+        flaky,
+        success,
+        inconclusive,
+        skipped,
+        unset
+    ).reversed()
```

Comment:
Why not just save this in the correct order?

Reply:
because of keeping visual order same as gcloud's https://github.com/Flank/gcloud_cli/blob/3c30bb59d18fa68c5a6df7d115786bc23f5fc224/google-cloud-sdk/lib/googlecloudsdk/api_lib/firebase/test/results_summary.py#L61